backend/app/api/endpoints/cloud_projects.py

```
@@ -120,6 +128,33 @@ def update_cloud_project(
     return _project_response(db, project, current_user)
 
 
+@router.post(
+    "/{project_id}/external-references",
+    response_model=ExternalReferenceResponse,
+)
+def register_external_reference(
+    project_id: int,
+    values: ExternalReferenceRegister,
+    db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)